.github/workflows/build-wheel-linux-x86_64.yaml

```diff
@@ -70,7 +70,7 @@ jobs:
               -DLLVM_ENABLE_ASSERTIONS=ON \
               -DLLVM_INSTALL_UTILS=ON
 
-        cmake --build llvm-build --target lld check-mlir -j$(nproc)
+        LIT_FILTER_OUT="Bytecode" cmake --build llvm-build --target lld check-mlir -j$(nproc)
```

Comment:
If everything passes, let's add a note here and there. 

We may need to update `mlir/Makefile` as well.

Reply:
https://github.com/PennyLaneAI/catalyst/pull/185/commits/f4c28d6719f87d61bd9007a8e96883eb025b3d56